conformance/metadata-fhirr4-service/README.md

```diff
@@ -1 +1,112 @@
+
+    A sample `Config.toml` is as below.
+
+    ```
+    ## server related configurables
+    [configFHIRServer]
+    version = "1.2.0"
+    name = "WSO2OpenHealthcareFHIR"
+    title = "FHIR Server"
+    status = "active"
+    experimental = true
+    date = "2022-11-24"
+    kind = "instance"
+    fhirVersion = "4.0.1"
+    format = ["json"]
+    patchFormat = ["application/json-patch+json"]
+    implementationUrl = "<FHIR_BASE_URL>"
+    implementationDescription = "WSO2 Open Healthcare FHIR"
+
+    ## server security related configurables
+    [configRest]
+    mode = "server"
+    resourceFilePath = "resources/fhir_resources.json"
+    interaction = ["search-system"]
+    [configRest.security]
+    cors = false
+    discoveryEndpoint = "https://api.asgardeo.io/t/<organization_name>/oauth2/token/.well-known/openid-configuration"
+    managementEndpoint = "https://api.asgardeo.io/t/<organization_name>/oauth2/manage"
+    ```
+
+3. Run the project.
+
+    ```ballerina
+    bal run
+    ```
+
+4. Invoke the APIs.
+
+    Sample request for FHIR Capability Statement:
+
+    ```
+    curl --location 'localhost:9090/fhir/r4/metadata'
+    ```
+
+## [Optional] Deploy in Choreo
+
+WSO2’s Choreo (https://wso2.com/choreo/) is an internal developer platform that redefines how you create digital experiences. Choreo empowers you to seamlessly design, develop, deploy, and govern your cloud native applications, unlocking innovation while reducing time-to-market. You can deploy the healthcare prebuilt services in Choreo as explained below. 
+
+### Prerequisites
+
+If you are signing in to the Choreo Console for the first time, create an organization as follows:
+
+1. Go to https://console.choreo.dev/, and sign in using your preferred method.
+2. Enter a unique organization name. For example, Stark Industries.
+3. Read and accept the privacy policy and terms of use.
+4. Click Create.
+This creates the organization and opens the Project Home page of the default project created for you.
+
+### Steps to Deploy Metadata Prebuilt Service in Choreo
+1. Create Service Component
+    * Fork the pre-built Ballerina services repository (https://github.com/wso2/open-healthcare-prebuilt-services) to your Github organization.
+    * Follow the official documentation to create and configure a service: https://wso2.com/choreo/docs/develop-components/develop-services/develop-a-ballerina-rest-api/#step-1-create-a-service-component.
+
+    * Once the component creation is complete, you will see the component overview page.
+
+2. Configure and Deploy
+
+    Follow the official documentation to deploy the Metadata prebuilt service to your organization https://wso2.com/choreo/docs/develop-components/develop-services/develop-a-ballerina-rest-api/#step-2-build-and-deploy.
+
+    On deployment configurables mentioned in [Configurations](#configurations) needs to be configured in Choreo configurable editor.
+
+### Configurations
```

Comment:
```suggestion
## Configurations
```

Reply:
Updated in https://github.com/wso2/open-healthcare-prebuilt-services/commit/b1b355e6d8a01038dca36588c1af3cd618f87fa1